t/05-scheduler-restart-and-duplicate.t

```diff
@@ -304,7 +304,6 @@ subtest 'restarting one of two independent root jobs (only related indirectly vi
     my $res = $root_1->auto_duplicate;
     is ref $res, 'OpenQA::Schema::Result::Jobs', 'no error when duplicating root1' or always_explain $res;
     my $cloned = $res->{cluster_cloned};
-    explain $cloned;
```

Comment:
After reading Tina's comment I looked up the docs of [explain/always_explain](https://metacpan.org/pod/Test::Most#always_explain-and-always_show). I forgot about the link to verbose mode.